tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/hlo_legalize_to_lhlo.cc

```diff
@@ -287,11 +287,33 @@ struct HloToLhloReduceOpConverter : public BaseOpConversion<mhlo::ReduceOp> {
   }
 };
 
-// Legalize mhlo.return to a lmhlo.copy and lmhlo.terminator. This functionality
-// is provided by mlir buffer assignment, so use the pattern from there.
-using HloToLhloReturnOpConverter =
-    BufferAssignmentReturnOpConverter<mhlo::ReturnOp, lmhlo::TerminatorOp,
-                                      lmhlo::CopyOp>;
+// Legalize mhlo.return to a lmhlo.copy and lmhlo.terminator.
+struct HloToLhloReturnOpConverter : public BaseOpConversion<mhlo::ReturnOp> {
+ public:
+  using BaseOpConversion<mhlo::ReturnOp>::BaseOpConversion;
+
+  LogicalResult matchAndRewrite(
+      mhlo::ReturnOp returnOp, ArrayRef<Value> operands,
+      ConversionPatternRewriter& rewriter) const final {
+    Location loc = returnOp.getLoc();
+    Block& entryBlock = returnOp.getParentRegion()->front();
```

Comment:
Google style (other than LLVM/MLIR) uses lower case like `entry_block`. Same for all other locals and function arguments.